demos/MLSys/demo.ipynb

```diff
@@ -0,0 +1,125 @@
+      "text/plain": [
+       "<drawSvg.raster.Raster at 0x7f84d868e310>"
+      ]
```

Comment:
Is this something that was generated? Is this something we want to commit? How do we make notebooks in a ready to commit format?

Reply:
I ran `nbstripout` on the notebook and it got rid of this